app/src/sharedTest/java/org/oppia/app/testing/FontScaleTestActivityTest.kt

```diff
@@ -0,0 +1,93 @@
+package org.oppia.app.testing
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.model.StoryTextSize
+import org.oppia.app.utility.FontSizeMatcher
+
+/** Tests for [FontScaleTestActivity]. */
+@RunWith(AndroidJUnit4::class)
+class FontScaleTestActivityTest {
```

Comment:
Please also add tests to verify that the exploration activity properly resizes font when the user enters & leaves it when the user sets their their font scale. This test suite is great for verifying the correctness of FontScaleConfigurationUtil, but doesn't verify that the exploration font size is correctly resized.

Reply:
@BenHenning this can be done after finalising the implementation of ExplorationManagerFragment?